source/features/dim-bots.css

```diff
@@ -23,23 +23,23 @@ Commits that have a long commit message will not be dimmed after the user uncoll
 ALL the following rules define the non-focused state
 */
 
-.rgh-dim-bot:not(.rgh-tagged, .navigation-focus, .Details--on) .mb-1, /* Commit titles, dim */
-.rgh-dim-bot:not(.rgh-tagged, .navigation-focus) .Box-row--drag-hide { /* PR row, dim */
+.rgh-dim-bot:not(.rgh-tagged, .navigation-focus, :hover, .Details--on) .mb-1, /* Commit titles, dim */
+.rgh-dim-bot:not(.rgh-tagged, .navigation-focus, :hover) .Box-row--drag-hide { /* PR row, dim */
 	opacity: 20%;
 	transition-delay: 0s;
 }
 
-.rgh-dim-bot:not(.rgh-tagged, .navigation-focus, .Details--on) .mb-1 ~ .d-flex,
-.rgh-dim-bot:not(.rgh-tagged, .navigation-focus, .Details--on) > .d-md-block {
+.rgh-dim-bot:not(.rgh-tagged, .navigation-focus, :hover, .Details--on) .mb-1 ~ .d-flex,
+.rgh-dim-bot:not(.rgh-tagged, .navigation-focus, :hover, .Details--on) > .d-md-block {
 	opacity: 20%;
 	margin-bottom: -1.6em;
 	visibility: hidden;
 	/* Delay visibility transition for a slightly shorter time than --rgh-dim-bots-delay, to make the animation smoother */
 	transition-delay: 0s, var(--rgh-dim-bots-delay), 9.9s;
```

Comment:
Can you also restore the 60s/1000s delay I had suggested a while back? 10s is not enough. It causes a cascade of collapsing items while I’m still using the list.

Reply:
I don't deal with bots a lot these days so I don't feel strongly, but I still think 10s makes a good default (or maybe just a little longer, 15s).

But actually this is "configurable":

```css
.rgh-dim-bots--after-hover {
	--rgh-dim-bots-delay: 60s;
}

.rgh-dim-bot:not(.rgh-tagged, .navigation-focus, :hover, .Details--on) .mb-1 ~ .d-flex,
.rgh-dim-bot:not(.rgh-tagged, .navigation-focus, :hover, .Details--on) > .d-md-block,
.rgh-dim-bot:not(.rgh-tagged, .navigation-focus, :hover) .labels,
.rgh-dim-bot:not(.rgh-tagged, .navigation-focus, :hover) .text-small:is(.color-text-secondary, .color-fg-muted) {
	transition-delay: 0s, var(--rgh-dim-bots-delay), 59.9s;
}
```

Reply:
I’ve seen it first hand, it’s not good UX, there’s no particular reason to collapse them again especially since we dim them already anyway. I’d default to “forever”: 1000s